app/experimenter/nimbus-ui/package.json

```diff
@@ -103,7 +103,30 @@
     "stylelint-config-prettier": "^8.0.2"
   },
   "eslintConfig": {
-    "extends": "react-app"
+    "extends": [
+      "react-app",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:prettier/recommended",
+      "plugin:react/recommended",
+      "prettier",
+      "prettier/@typescript-eslint",
+      "prettier/react"
+    ],
+    "rules": {
+      "@typescript-eslint/no-explicit-any": "off",
+      "@typescript-eslint/explicit-function-return-type": "off",
+      "@typescript-eslint/no-use-before-define": "off",
+      "@typescript-eslint/no-non-null-assertion": "off",
+      "@typescript-eslint/camelcase": "off",
+      "@typescript-eslint/ban-ts-ignore": "off",
+      "@typescript-eslint/no-empty-function": "off",
+      "react/display-name": "off",
+      "react/prop-types": "off",
+      "@typescript-eslint/no-unused-vars": "off",
+      "@typescript-eslint/explicit-module-boundary-types": "off",
+      "@typescript-eslint/ban-ts-comment": "off",
+      "import/no-anonymous-default-export": "off"
```

Comment:
These don't seem redundant with [the rules in eslint-config-react-app](https://github.com/facebook/create-react-app/blob/main/packages/eslint-config-react-app/index.js)? Weirdly, I picked up a few *new* lint issues that seemed legit, but not sure why we didn't see them before